plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/expression/RewriteIn.java

```diff
@@ -14,6 +14,7 @@
 package io.trino.plugin.jdbc.expression;
 
```

Comment:
https://github.com/trinodb/trino/pull/23455#issuecomment-2355709792

> also I see clickhouse connector can both expose `FixedString/String` as Trino `VARCHAR` and `VARBINARY` (default).
> 
> So do we need to test in both modes the pushdown? Does it matter?

@hashhar @Praveen2112 considering
> // TODO (https://github.com/trinodb/trino/issues/7102) reconsider default behavior

I think that support for `VARBINARY` may be considered as a separate effort.